specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/ContainerApps.json

```diff
@@ -856,6 +860,34 @@
         }
       }
     },
+    "Runtime": {
+      "description": "Container App Runtime configuration.",
+      "type": "object",
+      "properties": {
+        "kind": {
+          "description": "Container App language runtime kind",
+          "enum": [
+            "generic",
+            "java"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "kind",
+            "modelAsString": true
+          }
+        },
+        "java": {
```

Comment:
Could you please confirm that ARM will treat this object as optional, and that we won't have to declare it if the customer didn't define it?

Reply:
sure, no "required" restriction on runtime variables

Reply:
Thanks!